apps/backend-e2e/tsconfig.json

```diff
@@ -8,6 +8,5 @@
     }
   ],
   "compilerOptions": {
-    "esModuleInterop": true
   }
```

Comment:
Can probably removed completly, might be wrong tho.

Reply:
Some modules still require it